### PR TITLE
Retrieve match state

### PR DESCRIPTION
**Pull request by wstnmssr**
Add Get to the controller, implement method to retrieve from repo

### CODE REVIEW

**Review by aguentchev — APPROVED** [review]

File: xGame/Java/xgame/src/test/java/com/xgame/restservice/ChessMatchService.java [hunk 1]
```diff
@@ -135,4 +135,34 @@ void acceptInvite() {
...
+		catch(Exception e) {
+			fail();
+		}
+		finally {
+			//cleanup
+			matchRepo.deleteById(testMatchFromRepo.getId());
+			userRepo.delete(player1);
+			userRepo.delete(player2);
+		}
+	}
```
Comment (lancemcmuffin) [inline xGame/Java/xgame/src/test/java/com/xgame/restservice/ChessMatchService.java:168]: Could you also add a test for when no match is found and an exception is thrown?

**Review by lancemcmuffin — CHANGES_REQUESTED** [review]

File: xGame/Java/xgame/src/test/java/com/xgame/restservice/ChessMatchService.java (hunk 1 above)
Comment (tatunka) [reply xGame/Java/xgame/src/test/java/com/xgame/restservice/ChessMatchService.java:168]: I agree with Lance this is a good idea.

**Review by tatunka — APPROVED** [review]

File: xGame/Java/xgame/src/test/java/com/xgame/restservice/ChessMatchService.java (hunk 1 above)
Comment (wstnmssr) [reply xGame/Java/xgame/src/test/java/com/xgame/restservice/ChessMatchService.java:168]: Just added, let me know if you're looking for something different

**Review by aguentchev — APPROVED** [review]

File: xGame/Java/xgame/src/test/java/com/xgame/restservice/ChessMatchService.java (hunk 1 above)
Comment (tatunka) [reply xGame/Java/xgame/src/test/java/com/xgame/restservice/ChessMatchService.java:168]: Make it something dumb like -1 or 0. Something that can never occur in the database organically.

**Review by tatunka — CHANGES_REQUESTED** [review]
Just one super duper small thing...

**Review by tatunka — APPROVED** [review]

**Pull request merged**